src/cli.ts

```diff
@@ -7,7 +7,7 @@ import fs from 'node:fs';
 import path from 'node:path';
 import ora from 'ora';
 import ConfigStore from './config-store.js';
-import { API_URL, CONFIG_KEY } from './const.js';
+import { API_URL, CONFIG_KEY, DEFAULT_CLIENT_ID } from './const.js';
```

Comment:
Add a default client ID so it's more convenient for users to upload images without additional configuration. Also, a warning message will be displayed since it's possible that they will run into limit issue with this client ID